packages/flame_audio/lib/flame_audio.dart

```diff
@@ -49,9 +48,12 @@ class FlameAudio {
     String file,
     double volume,
     ReleaseMode releaseMode,
-    PlayerMode playerMode,
-  ) async {
+    PlayerMode playerMode, {
+    required AudioContext? audioContext,
```

Comment:
should we make the new parameter not-required?

Reply:
It's only used internally, wanted to make sure every method calling it, actually hands it over, so the dev is able to alter it, if needed.

Reply:
ah yes agree makes sense if it is internal!